app/views/govuk_publishing_components/components/_panel.html.erb

```diff
@@ -5,9 +5,9 @@
   <h2 class="govuk-panel__title">
     <%= title %>
   </h2>
-  <div class="govuk-panel__body">
-    <% if description %>
-      <%= description %>
-    <% end %>
-  </div>
+  <% if description %>
+    <div class="govuk-panel__body">
+      <%= description %> 
+    </div>
+  <% end %>
```

Comment:
This would also be useful to have an example.

Reply:
Also added an example here 🌮 🎉 👍